server/server.go

```diff
@@ -62,10 +63,13 @@ func NewServer(
 	analyzers map[string]lookout.Analyzer,
 	eventOp store.EventOperator,
 	commentOp store.CommentOperator,
+	organizationOp store.OrganizationOperator,
```

Comment:
IMO ~6 parameters are already many arguments to be passed to a function, simply because is annoying when reading to code to mentally keep track of everything that is being passed around. In this case maybe we could simply pass a struct of operators. You made me recalled Scala's standard library where you can define functions up to 22 parameters 😂 (https://www.scala-lang.org/api/2.12.1/scala/Function22.html).

Reply:
Passing a `conf` struct seems to be the better way.
On the other hand,
```go
func NewWatever(a, b, c, d, e, f, g, h, y type) Watever {
    return Watever{a, b, c, d, e, f, g, h, y}
}
```
seems a bit weird, since `NewWatever` func is doing nothing but pass arguments to the inside.

Reply:
If `NewWhatever` doesn't need to do anything particular, then it means that we could simply `Whatever{a, b, c, d, e, f, g, h, y}` from outside the package.

Reply:
Just remember guys that `Whatever{...}` wouldn't create compiling error as `NewWhatever(...)` when you add one more argument.